libdirections-onboard/src/main/java/com/mapbox/navigation/route/onboard/internal/MapboxOnboardRouter.kt

```diff
@@ -42,45 +43,31 @@ import kotlinx.coroutines.withContext
  */
 @MapboxModule(MapboxModuleType.NavigationOnboardRouter)
 class MapboxOnboardRouter(
+    private val applicationContext: Context,
     private val accessToken: String,
     private val navigatorNative: MapboxNativeNavigator,
     private val options: OnboardRouterOptions,
     private val logger: Logger,
     private val skuTokenProvider: SkuTokenProvider
 ) : Router {
 
-    private companion object {
-        private const val TAG = "MapboxOnboardRouter"
+    internal companion object {
+        internal val loggerTag by lazy { Tag("MapboxOnboardRouter") }
```

Comment:
Do we want to use same `loggerTag` in `OnboardRouterFiles`? 🤔

Reply:
what's your opinion? am not sure if we want to use the logger to create feature tags

with the freedom to specify a tag, compelled to experiment 🙃 . what's the reason for specifying the tag again? why not automatically populate it with the class name like Timber does?

Reply:
without a tag it uses  I/MapboxLogger: Initial size is 3488 bytes

so want to have "OnboardRouter" somewhere. but best to avoid anything new in special, will grab the class name